### PR TITLE
Add test AD unit IDs to README

### PR DESCRIPTION
**Pull request by anton-kapralov**

### CODE REVIEW

**Review by zdyn — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ For any other questions, please email sdk@liftoff.io.
       - [Objective-C](#objective-c-1)
     - [GDPR/CCPA and User Consent](#gdprccpa-and-user-consent)
   - [Creating a MoPub Custom SDK Network](#creating-a-mopub-custom-sdk-network)
+  - [Using test AD units](#using-test-ad-units)
```
[the remark this answers is not in the document]
Comment (anton-kapralov) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -560,6 +561,18 @@ Both banner and mrect ads use the `LiftoffBannerCustomEvent` class.
 
 ![](https://user-images.githubusercontent.com/573865/96619293-d6afe200-12ba-11eb-8a14-133be3d8f775.png)
 
+### Using test AD units
+
+For the test purposes you may want to consider using special test AD unit IDs
+that are automatically added to every app:
```
[the remark this answers is not in the document]
Comment (anton-kapralov) [reply README.md]: Done.

File: README.md [hunk 3]
```diff
@@ -560,6 +561,18 @@ Both banner and mrect ads use the `LiftoffBannerCustomEvent` class.
...
+
+| Ad Unit ID                        | Type         | Supported Sizes                      | Creative Type              |
+| --------------------------------- | ------------ | ------------------------------------ | -------------------------- |
+| `liftoff-banner-mrect-test`       | Banner       | 320x50, 728x90, 300x250              | `vast_video`, `html_video` |
+| `liftoff-interstitial-video-test` | Interstitial | 320x480, 480x320, 768x1024, 1024x768 | `vast_video`               |
+| `liftoff-interstitial-html-test`  | Interstitial | 320x480, 480x320, 768x1024, 1024x768 | `html_video`               |
+| `liftoff-rewarded-video-test`     | Rewarded Interstitial | 320x480, 480x320, 768x1024, 1024x768 | `vast_video`      |
```
[the remark this answers is not in the document]
Comment (anton-kapralov) [reply README.md]: Done.

File: README.md [hunk 4]
```diff
@@ -560,6 +561,18 @@ Both banner and mrect ads use the `LiftoffBannerCustomEvent` class.
...
 ![](https://user-images.githubusercontent.com/573865/96619293-d6afe200-12ba-11eb-8a14-133be3d8f775.png)
 
+### Using test AD units
+
+For the test purposes you may want to consider using special test AD unit IDs
+that are automatically added to every app:
+
+| Ad Unit ID                        | Type         | Supported Sizes                      | Creative Type              |
+| --------------------------------- | ------------ | ------------------------------------ | -------------------------- |
+| `liftoff-banner-mrect-test`       | Banner       | 320x50, 728x90, 300x250              | `vast_video`, `html_video` |
```
[the remark this answers is not in the document]
Comment (anton-kapralov) [reply README.md]: Done.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (anton-kapralov) [reply README.md]: Done.

File: README.md [hunk 5]
```diff
@@ -531,6 +532,18 @@ LOPrivacySettings.setHasUserConsent(true)
 [LOPrivacySettings setHasUserConsent:true];
 ```
 
+#### Test Ad Units
+
+Use the following ad unit IDs to display a LiftoffAds test creative and verify
+successful integration.
+
+| Ad Unit ID                        | Type           | Creative Type              |
```
Comment (zdyn) [inline README.md:540]: Can you s/Type/Size and s/Creative Type/Type to mirror the usage earlier in this README (see the Overview section)?

**Review by zdyn — APPROVED** [review]
Thanks, this will make future integrations a lot easier. BTW, in the future no need to respond to each individual comment. I'll assume that a review re-request means they were all addressed. If you want a way to track addressed comments for yourself, the 👍 emoji works (and doesn't send an email).

**Comment by anton-kapralov** [comment]
Thanks!

**Pull request merged**